src/app/pages/auth/auth.component.html

```diff
@@ -0,0 +1,91 @@
+<div class="reg">
+  <form [formGroup]="loginForm" (ngSubmit)="onSubmit()" class="reg__form">
+    <h1 class="reg__title">Enter in your account</h1>
+    <div class="reg__inputs">
+      <input
```

Comment:
move input in shared component, make this component multi functionality, and replace all input